src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsEvolveOasis.java

```diff
@@ -8,42 +8,42 @@
 
 import org.testng.annotations.Test;
 
-public class TestEvolveAds extends TemplateNoFirstLoad {
+public class TestAdsEvolveOasis extends TemplateNoFirstLoad {
 
   @GeoEdgeBrowserMobProxy(country = "CA")
   @UseUnstablePageLoadStrategy
   @Test(
       dataProviderClass = AdsDataProvider.class,
-      groups = {"TestEvolveAds"},
+      groups = "AdsEvolveOasis",
       dataProvider = "evolveTestPage"
   )
-  public void testEvolveCall_CA(String wikiName, String article) {
-    String testedPage = urlBuilder.getUrlForPath(wikiName, article);
-    AdsEvolveObject wikiPage = new AdsEvolveObject(driver, testedPage);
-    wikiPage.verifyEvolveCall();
+  public void adsEvolveOasisCA(String wikiName, String article) {
+    adsEvolveOasis(wikiName, article);
   }
 
   @GeoEdgeBrowserMobProxy(country = "AU")
```

Comment:
it could be interesting to move the country codes (abbreviations) to some class, where the values could be accessed through the full country name like: countryCode.Australia. This way it would improve the understanding of test and enhance the maintenance